_delphi_utils_python/data_proc/geomap/geo_data_proc.py

```diff
@@ -531,6 +533,56 @@ def derive_zip_to_state_code():
     )
 
 
+def derive_fips_hhs_crosswalk():
+    """
+    Builds a crosswalk between FIPS county codes and HHS regions.
+    """
+    if not isfile(join(OUTPUT_DIR, STATE_HHS_OUT_FILENAME)):
+        create_state_hhs_crosswalk()
+    if not isfile(join(OUTPUT_DIR, FIPS_POPULATION_OUT_FILENAME)):
+        create_fips_population_table()
+
+    fips_pop = pd.read_csv(
+        join(OUTPUT_DIR, FIPS_POPULATION_OUT_FILENAME), dtype={"fips": str, "pop": int}
+    )
+    state_hhs = pd.read_csv(
+        join(OUTPUT_DIR, STATE_HHS_OUT_FILENAME),
+        dtype={"state_code": str, "state_id": str, "state_name": str},
+    )
+
+    fips_pop["state_code"] = fips_pop["fips"].str[:2]
+    (
+        fips_pop.merge(state_hhs, on="state_code", how="left")
+        .drop(columns=["state_code", "pop"])
+        .to_csv(join(OUTPUT_DIR, FIPS_HHS_FILENAME), index=False)
+    )
+
+
+def derive_zip_hhs_crosswalk():
+    """
+    Builds a crosswalk between zip code and HHS regions.
+    """
+    if not isfile(join(OUTPUT_DIR, STATE_HHS_OUT_FILENAME)):
+        create_state_hhs_crosswalk()
+    if not isfile(join(OUTPUT_DIR, ZIP_STATE_CODE_OUT_FILENAME)):
+        derive_zip_to_state_code()
+
+    zip_state = pd.read_csv(
+        join(OUTPUT_DIR, ZIP_STATE_CODE_OUT_FILENAME),
+        dtype={"zip": str, "pop": int, "state_code": str}
+    )
+    state_hhs = pd.read_csv(
+        join(OUTPUT_DIR, STATE_HHS_OUT_FILENAME),
+        dtype={"state_code": str, "state_id": str, "state_name": str},
+    )
+
+    (
+        zip_state.merge(state_hhs, on="state_code", how="left")
+        .drop(columns=["state_code", "state_id", "state_name"])
```

Comment:
Update to reflect changed column names on line 576. Should be the same as line 556.

Reply:
In this case, the zip_state file is the one introducing these columns. The other function uses the fips population file (was just copying an different fips -> * example) which only has the population column to drop

Reply:
Got it.